client/js/controllers/game.js

```diff
@@ -125,7 +125,47 @@ angular.module('mean.system')
         }
       };
 
-      $scope.winnerPicked = () => game.winningCard !== -1;
+      $scope.winnerPicked = () => game.winningCard !== 1;
+
+      $scope.startGame = () => {
+        game.startGame();
+      };
+
+      $scope.abandonGame = () => {
+        game.leaveGame();
+        $location.path('/');
+      };
+      $scope.shuffleCards = () => {
+        const card = $(`#${event.target.id}`); //eslint-disable-line
+        card.addClass('animated flipOutY');
+        setTimeout(() => {
+          $scope.startNextRound();
+          card.removeClass('animated flipOutY');
+          $('#start-modal').modal('hide');
```

Comment:
why start modal, we already have a start-modal, shouldn't this be draw-card modal or another name?